src/app/personside/infotabs/meldinger/MeldingerContainer.tsx

```diff
@@ -53,11 +53,14 @@ function MeldingerContainer(props: MeldingerDyplenkeRouteComponentProps) {
     const [sokeord, setSokeord] = useState('');
     const traaderEtterSok = useSokEtterMeldinger(hasData(traaderResource) ? traaderResource.data : [], sokeord);
 
-    useEffect(() => {
-        if (traadIUrl && traaderEtterSok.length > 0 && !traaderEtterSok.includes(traadIUrl)) {
-            props.history.push(dyplenker.meldinger.link(traaderEtterSok[0]));
-        }
-    }, [traadIUrl, traaderEtterSok, props.history, dyplenker.meldinger]);
+    useEffect(
+        function setterForsteSokeresultatTilDyplenke() {
```

Comment:
`visFørsteTreffITraadvisning`?

Navnet slik det er nå forklarer hvordan istedenfor hva (imperrativt vs deklarativt). Prøv å gjør navne deklarative, vi er først og fremst interessert i hva en funksjon gjør, ikke hvordan. La oss feks si at vi i fremtiden endrer logikken og ikke lenger bruker dyplenker men feks redux til å bestemme hvilken melding som vises. Da må vi isåfall huske å endre metodenavn til feks `leggerTraadIRedux`. Det vil fortsatt være vanskelig å skjønne hva som var den orginale oppgaven/motivatsjonen for å skrive funksjonen og vi endret navn selv om oppgaven egentlig er den samme. Prøv å fortelle hva som er oppgaven/motivatsjonen til funksjonen i navnet.

Reply:
Jeg er helt enig! Godt innspill. Ble mer en forklaring av hvordan metoden gjør istedenfor hva utfallet av metoden er!